scripts/travisci_run.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
 set -eux
 
+# Install go 1.5
+eval "$(gimme 1.5)"
```

Comment:
It would be prefered if you did this in the `.travis.yml` file where we also setup Android: https://github.com/facebook/buck/blob/master/.travis.yml#L18